plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -695,6 +695,81 @@ public void testIncrementalStatisticsUpdateOnInsert()
                         "(null, null, null, null, 75.0, null, null)");
     }
 
+    @Test
+    public void testAnalyzeWhenNewColumnAdded()
+    {
+        String tableName = "test_statistics_on_insert_when_new_column_added_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (col_int_1 bigint, col_varchar_1 varchar)");
+
+        assertUpdate(
+                disableStatisticsCollectionOnWrite(getSession()),
+                "INSERT INTO " + tableName + " VALUES (11, 'aa')", 1);
+
+        assertUpdate("ANALYZE " + tableName);
+
+        assertQuery(
+                "SHOW STATS FOR " + tableName,
+                """
+                        VALUES
+                        ('col_int_1', null, 1.0, 0.0, null, 11, 11),
+                        ('col_varchar_1', 2.0, 1.0, 0.0, null, null, null),
+                        (null, null, null, null, 1.0, null, null)
+                        """);
```

Comment:
Technically, this is redundant. we don't have to test that ANALYZE works in a simple case.
This is already covered by other existing tests